src/Avalonia.Controls/TabItem.cs

```diff
@@ -62,24 +58,6 @@ public bool IsSelected
 
         protected override AutomationPeer OnCreateAutomationPeer() => new ListItemAutomationPeer(this);
 
-        protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
-        {
-            base.OnAttachedToVisualTree(e);
-
-            _ownerSubscriptions?.Dispose();
-            _ownerSubscriptions = null;
-
-            if (this.FindAncestorOfType<TabControl>() is { } owner && owner.IndexFromContainer(this) != -1)
-            {
-                SubscribeToOwnerProperties(owner);
-            }
-        }
-
-        protected void SubscribeToOwnerProperties(AvaloniaObject owner)
```

Comment:
Removal of this method is a breaking change. As noticed from the CI fail.
Can yo keep this method empty with Obsolete attribute? Just in case.

Reply:
@maxkatz6 done